website/docs/guides/best-practices/how-we-mesh/mesh-3-implementation.md

```diff
@@ -0,0 +1,130 @@
+
+- When you’ve **confirmed the right groups**, it's time to split your projects.
+  - **Do *one* group at a time**!
+  - **Do *not* refactor as you migrate**, however tempting that may be. Focus on getting 1-to-1 parity and log any issues you find in doing the migration for later. Once you’ve fully migrated the project then you can start optimizing it for its new life as part of your mesh.
+- Start by splitting your project within the same repository for full git tracking and easy reversion if you need to start from scratch.
+
+
+## Connecting existing projects
+
+Some organizations may already be coordinating across multiple dbt projects. Most often this is via:
+
+1. Installing parent projects as dbt packages
+2. Using `{{ source() }}` functions to read the outputs of a parent project as inputs to a child project. 
+
+This has a few drawbacks:
+
+1. If using packages, each project has to include *all* resources from *all* projects in its manifest, slowing down dbt and the development cycle.
+2. If using sources, there are breakages in the lineage, as there's no real connection between the parent and child projects.
+
+The migration steps here are much simpler than splitting up a monolith!
+
+1. If using the `package` method:
+   1. In the parent project:
+      1. mark all models being referenced downstream as `public` and add a model contract.
+   2. In the child project:
+      1. Remove the package entry from `packages.yml`
+      2. Add the upstream project to your `dependencies.yml`
+      3. Update the `{{ ref() }}` functions to models from the upstream project to include the project name argument.
+1. If using `source` method:
+   1. In the parent project:
+      1. mark all models being imported downstream as `public` and add a model contract.
+   2. In the child project:
+      1. Add the upstream project to your `dependencies.yml`
+      2. Replace the `{{ source() }}` functions with cross project `{{ ref() }}` functions.
+      3. Remove the unnecessary `source` definitions.
+
+## Additional Resources
+### Our example projects
+
+We've provided a set of example projects you can use to explore the topics covered here. We've split our [Jaffle Shop](https://github.com/dbt-labs/jaffle-shop) project into 3 separate projects in a multi-repo dbt Mesh. Note that you'll need to leverage dbt Cloud to use multi-project architecture, as cross-project references are powered via dbt Cloud's APIs.
+
+- **[Platform](https://github.com/dbt-labs/jaffle-shop-mesh-platform)** - containing our centralized staging models.
+- **[Marketing](https://github.com/dbt-labs/jaffle-shop-mesh-marketing)** - containing our marketing marts.
+- **[Finance](https://github.com/dbt-labs/jaffle-shop-mesh-finance)** - containing our finance marts.
+
+### dbt-meshify
+
+We recommend using the `dbt-meshify` [command line tool](<https://dbt-labs.github.io/dbt-meshify/0.1/>) to help you do this. This comes with CLI operations to automate most of the above steps.
```

Comment:
by CLI, do we mean dbt Cloud CLI or the command line interface in general?

Reply:
command line interface in general - dbt-meshify is separate from the Cloud CLI

Reply:
We've been moving away from CLI except when using Cloud CLI, so we might want to do the same here